packages/webservice/README.md

```diff
@@ -95,6 +95,12 @@ dependencies {
 
 To run the demo app
 
+```shell script
+$ ./gradlew bootRun -Pbuild_env=prod
```

Comment:
Given
https://github.com/yahoo/yavin/pull/1232/files#diff-4da8fcac60edb2f04baba02c1236a1d9978d511bbcffd2f638ca4aa729786117R6

Should this be:
```suggestion
$ ./gradlew bootRun -Penvironment=prod
```
??

Reply:
i think we wanted `./gradlew bootRun -Penvironment=production` 
instead

Reply:
```suggestion
$ ./gradlew bootRun -Penvironment=production
```